examples/react-instantsearch/src/App.js

```diff
@@ -29,6 +29,8 @@ const Hits = connectHits(
           <button
             onClick={() => {
               window.aa('click', {
+                eventName: "hit-clicked",
+                indexName: process.env.INDEX_NAME,
```

Comment:
We should avoid to document this usage because the widget can be scoped inside an `Index` component that change the index currently targeted. We should document the "best" way to access all those information.

Reply:
I'll have to get rid of process.env.INDEX_NAME in the close future
The ultimate best way to use insights client will come with the instantsearch integration.
I filed an issue for that